jsonschema-validation.xml

```diff
@@ -1015,8 +1031,9 @@
 
             <section title="contentSchema">
                 <t>
-                    If the instance is a string, and if "contentMediaType" is present, this
-                    property contains a schema which describes the structure of the string.
+                    If the instance is a string or an untyped binary resource,
```

Comment:
What about: "If the instance is a string or an untyped binary resource having an application-defined mapping to the data model"